### PR TITLE
Remove peerDependencies

### PR DESCRIPTION
**Pull request by kevinetienne**
`npm > 3` use `peerDependencies` as a check at the end of the install, error is:

``` bash
 npm WARN EPEERINVALID grunt-gettext-checker@3.0.0 requires a peer of grunt@~0.4.0 but none was installed.
    npm WARN EPACKAGEJSON teva-becon@0.0.0 No license field.
    npm ERR! Linux 3.19.0-58-generic
    npm ERR! argv "/home/travis/.nvm/versions/node/v5.0.0/bin/node" "/home/travis/.nvm/versions/node/v5.0.0/bin/npm" "install"
    npm ERR! node v5.0.0
    npm ERR! npm  v3.3.6

    npm ERR! Callback called more than once.
```

### CODE REVIEW

**Comment by KevinEtienne** [comment]
@wytrych @henrahmagix can you review/merge please?

File: package.json [hunk 1]
```diff
@@ -31,9 +31,6 @@
     "jit-grunt": "0.3.2",
     "time-grunt": "0.3.1"
   },
-  "peerDependencies": {
-    "grunt": "~0.4.0"
```
Comment (henrahmagix) [inline package.json]: Instead, upgrade to `>=0.4.0`. This will match most environments and not cause an error

**Comment by henrahmagix** [comment]
You do need `grunt` to use this plugin, so it's a sensible peer dependency.

Since grunt v1 came out, the Grunt team made a bot that searched popular grunt plugin repos for the peer dependency and automatically made a pull-request to upgrade it to `>=` to allow installing alongside grunt v1. So I think we should do the same =)

File: package.json (hunk 1 above)
Comment (henrahmagix) [inline package.json]: At that point, the change would no longer be breaking, so it can be a semver PATCH upgrade

Comment (KevinEtienne) [inline package.json]: Will the error be just different? Eg.

```bash
 npm WARN EPEERINVALID grunt-gettext-checker@3.0.0 requires a peer of grunt@>=0.4.0 but none was installed.
```

Comment (henrahmagix) [inline package.json]: Yes, but `grunt` should be installed to use this, so it's a useful error =)

**Pull request closed without merging**